meilisearch-release-changelog/v1.15.0.md

```diff
@@ -0,0 +1,63 @@
+# v1.15.0 release changelogs
+
+<!-- The following line should ONLY be put PRE-release changelogs -->
+> [!WARNING]
+> Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Is something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+Meilisearch v1.15.0 introduces improves the indexing performances and dumpless experience.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+Some SDKs might not include all new features. Consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+# New features and updates 🔥
+
+## Deactivate Typo Tolerance on Numbers
+
+TBD
+
+Done by @ManyTheFish in [#5494](https://github.com/meilisearch/meilisearch/pull/5494).
+
+## Make it Possible to Rollback Dumpless Upgrades
+
+TBD
+
+Done by @dureuill in [#5523](https://github.com/meilisearch/meilisearch/pull/5523).
+
+## Other improvements
```

Comment:
Is this the definitive order of the list below? I think we want to promote the most useful (for users!) first.

Reply:
Reordered by putting:
1) new features/abilities
2) new informational fields
3) Error messages enhancements
4) experimental related changes